src/index.html

```diff
@@ -9,6 +9,94 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header__box">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a
+              href="#Apple"
+              class="is-active a__link"
+            >
+              Apple
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              href="#Apple"
              class="is-active nav__link"
            >
              Apple
```

ul - unordered list - nav__list
li - list item - nav__item
tag a - link - nav__link

simple names - better understanding and reusability